sykmeldinger-backend-proxy/routes.yaml

```diff
@@ -0,0 +1,8 @@
+GET:
```

Comment:
Betyr dette at det kun er bestemte metoder til bestemte ruter som slipper gjennom? Det er jo gull, for jeg ser at vi mottar innmari mye rart mot blant annet syfosmregister i dag 🏆 

Men, gjør ikke frontend typisk en OPTIONS før den gjør POST? Sånn preflight-greier..?

Reply:
frontend gjør det, men express appen håndterer det https://github.com/navikt/flex-proxy/blob/master/flex-proxy/src/server.ts